app/lib/pybot_slack_client.rb

```diff
@@ -0,0 +1,32 @@
+class PybotSlackClient
+  include HTTParty
+
+  base_uri ENV['SLACK_SERVICE_URL']
+
+  attr_reader :headers
+
+  def initialize
```

Comment:
In general we should put logging around these requests too. I want to be able to open up the logs and see what's happening for when we have failures. Something like sending request, and logging the request response in addition to everything else should be good.

Reply:
Do you want it to raise an error if the `response.status != 200` and or if `response.ok === 'fail'` in response?

Reply:
Per allen:
```json
{
    "ok": false,
    "error": "already_in_team"
}
```
returns a 200 from pybot service. For now let's just throw an error if status != 200, then we log them all and we can add cases later.